city_scrapers/spiders/chi_ssa_35.py

```diff
@@ -0,0 +1,144 @@
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa35Spider(CityScrapersSpider):
+    name = "chi_ssa_35"
+    agency = "Chicago Special Service Area #35 Lincoln Ave"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://www.lincolnparkchamber.com/"
+        + "businesses/special-service-areas/lincoln-avenue-ssa/ssa-administration/"
+    ]
+
+    def parse(self, response):
+        _year = None
+        _type = ""
+        _description = ""
+
+        content_div = response.css("div.content_block.content.background_white")
+        inner_elements = content_div.css("h4, li, p")
+
+        for element in inner_elements:
+            meeting_content = {}
+
+            if "<h4>" in element.get():
+                # Type here could be Schedule, Agendas or Minutes
+                title = element.css("::text").get().split()
+                _type = title[-1]
+                try:
+                    _year = int(title[0])
+                except ValueError:
+                    _year = None
+
+            if "<p><em>" in element.get():
+                # The encode part here is to remove \xa0 and other characters like that
+                _description = (
+                    element.css("::text").get().encode("ascii", "ignore").decode()
+                )
+
+            if "<li>" in element.get():
+                meeting_content["date"] = element.css("::text").get()
+                if "href" in element.get():
+                    meeting_content["url"] = element.css("a::attr(href)").get()
+
+            if "<p><a" in element.get():
+                meeting_content["date"] = element.css("::text").get()
+                meeting_content["url"] = element.css("a::attr(href)").get()
+
+            meeting_content["description"] = _description
+            meeting_content["type"] = _type
```

Comment:
It looks like this isn't currently being used?

Reply:
It isn't. I was looking for somewhere to put this info.